oper8/test_helpers/kub_mock.py

```diff
@@ -312,7 +312,9 @@ def _add_handlers_for_kind(self, namespace, kind, api_version):
                 endpoint = f"{api_endpoint}/{kind_variant}/*"
 
             log.debug2(
-                "Adding POST & GET & WATCH & PATCH handler for (%s: %s)", kind, endpoint
+                "Adding POST & PUT &GET & WATCH & PATCH handler for (%s: %s)",
```

Comment:
NIT: Looks like you lost a space here?
```suggestion
                "Adding POST & PUT & GET & WATCH & PATCH handler for (%s: %s)",
```